mobile/src/components/ButtonGroup.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { Text, StyleSheet, View, TouchableOpacity } from 'react-native';
+import { Colors } from '../styles/Colors';
+import { TextStyles } from '../styles/TextStyles';
+
+// button labels and callback function for passing key of active button to parent
+type ButtonGroupProps = { btn1: string, btn2: string, btn3: string, onBtnChange };
+
+/**
+ * Renders an inline group of three button tabs
+ */
+export const ButtonGroup: React.FC<ButtonGroupProps> = ({ btn1, btn2, btn3, onBtnChange }) => {
```

Comment:
I'm not sure how useful this component is. I definitely think we should have components for individual buttons (maybe <SmallButton> and <BigButton>) that match the style guide. I think once you have a component for an individual button it should be easy enough to just define 3 of them in your book details screen and space them manually.

If you decide a ButtonGroup is still helpful though, I think it would be better to take an array of (any number of) buttons, hardcoding it to three buttons is just going to restrict us down the line.